sdk/identity/azure-identity-v2/src/main/java/com/azure/identity/v2/implementation/util/IdentityUtil.java

```diff
@@ -26,6 +32,17 @@ public final class IdentityUtil {
     public static final HttpHeaderName X_TFS_FED_AUTH_REDIRECT = HttpHeaderName.fromString("X-TFS-FedAuthRedirect");
     public static final HttpHeaderName X_VSS_E2EID = HttpHeaderName.fromString("x-vss-e2eid");
     public static final HttpHeaderName X_MSEDGE_REF = HttpHeaderName.fromString("x-msedge-ref");
+    public static final String PROPERTY_AZURE_TENANT_ID = "AZURE_TENANT_ID";
```

Comment:
Previously we had a lot of these in Core. I guess we aren't putting these in corev2 but wanted to double check. @alzimmermsft?

Reply:
In Core, we started putting them there and then stopped midway.
If we get demand for these, we can put them in core v2 down the road.

Reply:
I agree with @g2vinay, let's keep this in Identity for now and if needed we can promote this to Core.